src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -235,7 +235,7 @@ public PortableInfoboxObject isVideoCaptionVisible() {
     return this;
   }
 
-  public PortableInfoboxObject isHeroImageCentered() {
```

Comment:
There was also project called hero image related to main pages. The name hero image confuses me and will confuse anyone who know about the project. The image in infobox looks exactly as any other normal/typical image. Why should it be called hero image ? What makes it different from other images that appear in other pages ?